Lib/subprocess.py

```diff
@@ -978,6 +979,18 @@ def __init__(self, args, bufsize=-1, executable=None,
 
             raise
 
+    def __repr__(self):
+        max_args_length = 80
```

Comment:
This is better, but now the repr could be very long:

```python
>>> len(f'<Popen: returncode:None args:{"a"*80}>')
110
```

So either replace 80 with 50, or simplify like so:
```python
obj_repr = (
    f"<{self.__class__.__name__}: "
    f"returncode: {self.returncode} args: {args}>"
)
if len(obj_repr) > 80:
    obj_repr = obj_repr[:76] + "...>"